### ISSUE TITLE
L900 failed set up

### ISSUE THREAD

**Issue by andrewgarrard** [petretiandrea/home-assistant-tapo-p100#587]
Integration reports  "failed set up, will try". Tried rebooting, restarting, removing and reading integration and still fails. I can't see anything obvious in the logs:
"Config entry 'back door light' for tapo integration not ready yet: None; Retrying in background"
When I try and add the integration again, it says "invalid authentication" (the username and password and up are correct)..

Sorry, bit of a noob.

**Comment by petretiandrea** [comment]
Have you 2FA enabled?

**Comment by andrewgarrard** [comment]
Thanks for the reply. I don't have 2FA enabled - I assume that would make security settings more complex and deliberately didn't enable it. Out of frustration, I ended up wiping and reinstalling the whole OS. I was previously running in a container but decided that it would be cleaner to install hassio as the OS on the card would be cleaner and have fewer bugs. I managed to get the Tapo integration working on that - thanks for building it and it must have been something about my installation.  

I'm getting another little bug where the brightness is defaulting back to 100%.  Even if I edit the text in the scenes file, when I activate them it always defaults to 100%. This is on a new, updated, clean installation. Should I start another thread as it is a new problem?
